etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -267,11 +275,50 @@ def load(cls, path: pathlib.Path) -> "TS2VecEmbeddingModel":
         path:
             Path to load object from.
 
+            - if `path` is not None and `model_name` is None, load the local model from `path`.
```

Comment:
Let's wrap `path` and `model_name` in double ``. This also holds for other lines.